samples/04.3-open-telemetry/README.md

```diff
@@ -0,0 +1,60 @@
+# Visualize tracing of connectors with Open Telemetry and Jaeger
+
+This sample builds on top of [sample 04.0-file-transfer](../04.0-file-transfer) to show how you can generate traces with Open Telemetry and collect and visualize them with Jaeger.
+
+We will use the Open Telemetry java agent. It dynamically injects bytecode to capture telemetry from several popular [libraries and frameworks](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation).
+
+In order to visualize and analyze the traces, we need to use an [Open Telemetry exporter](https://opentelemetry.io/docs/instrumentation/js/exporters/) to export data into the  [Jaeger](https://www.jaegertracing.io/) tracing backend.
+
+## Prerequisites
+
+Download the [opentelemetry-javaagent.jar](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases) and place it in the root folder of this sample.
```

Comment:
version has to match exactly the one in the code

Reply:
I will replace the link with the concrete one for v1.10.1